packages/files-ui/src/Contexts/FilesContext.tsx

```diff
@@ -662,7 +662,10 @@ const FilesProvider = ({ children }: FilesContextProps) => {
       return Promise.resolve()
     } catch (error: any) {
       console.error(error)
-      let errorMessage = `${t`An error occurred: `} ${typeof(error) === "string" ? error : error.error.message ? error.error.message : ""}`
+      let errorMessage = `${t`An error occurred: `} ${typeof(error) === "string"
+        ? error : error.message
+          ? error.message : error.error.message
+            ? error.error.message : ""}`
```

Comment:
Would something like this work
```suggestion
      let errorMessage = `${t`An error occurred: `} ${error || error.message || error.error.message || ""}
```

This suggestion still looks awful to my eyes, but a little bit better than 3 ternaries.

Reply:
Yeah was just following the existing approach, this is much nicer